backend/scripts/health_check.py

```
@@ -0,0 +1,410 @@
+        logger.info("🔍 Testing multi-tenant operations...")
+
+        try:
+            from app.core.database import get_db_session
+            from app.repositories.tenant import TenantRepository
+            from app.models.tenant import Tenant
+            import uuid
+
+            async with get_db_session() as session:
+                # Create a temporary tenant for testing
+                test_tenant_id = uuid.uuid4()
+                tenant_repo = TenantRepository(session, test_tenant_id)
```

Ruff (0.13.1)

Local variable `tenant_repo` is assigned to but never used

Remove assignment to unused variable `tenant_repo`

(F841)